extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -428,7 +428,10 @@ function _mapDisplaySets(
       };
 
       if (componentType === 'thumbnailNoImage') {
```

Comment:
I see that we have a few references to these values. If it's not a lot of work we should add a new enum for ThumbnailType ('thumbnail', 'thumbnailTracked', 'thumbnailNoImage')

Reply:
I only changing this file because of the reject function